tests/rptest/tests/pandaproxy_test.py

```diff
@@ -7,10 +7,11 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0
 
+import http.client
```

Comment:
this could be split across 4 commits.

1. logging fixes
2. string interpolation
3. extract useful functions
4. dont auto create topics